### DISCUSSION TITLE
[RBAC] Access to node <node> denied, deny rule in auditor matched

### DISCUSSION THREAD

**Discussion by chriswachira** [gravitational/teleport#7102] [Q&A] [answered]
What could be causing the above error?

I had configured Teleport and it worked perfectly, for a few minutes. The nodes disappeared from the UI and tailing the Teleport service I noted that error.

![Screenshot_20210521_035234](https://user-images.githubusercontent.com/19710961/120068288-6d7e2180-c088-11eb-9fe0-4968056bc21b.png)

**Comment by klizhentas** [comment, marked as answer]
Hey 👋  https://github.com/gravitational/teleport/blob/master/lib/services/presets.go#L104 auditor role owner is denied to initiate active sessions, if you remove the role from user the session will be allowed.
